consensus/polybft/statesyncrelayer/state_sync_relayer.go

```diff
@@ -110,24 +109,33 @@ func (r *StateSyncRelayer) AddLog(log *ethgo.Log) {
 	if commitEvent.Match(log) {
 		vals, err := commitEvent.ParseLog(log)
 		if err != nil {
-			panic(err)
+			r.logger.Info("Failed to parse log", "err", err)
```

Comment:
Should we change the log level to error perhaps?

```suggestion
			r.logger.Error("Failed to parse log", "err", err)
```

Reply:
Sure, I left it like that to check in test logs